### ISSUE TITLE
Audio Jittering while recording on any mobile browser/FireFox cant record at all

### ISSUE THREAD

**Issue by zigfighter** [RocketChat/Rocket.Chat#14453]
On any mobile browser recording audio in a room while the chat is bussy, the audio stutters and skips. Alone in a fresh room thers no problem, but in rooms with f.e. 26k Messages in and swiping the channels list while recording, the audio is heavily skipping. Checked all browsers(chrome, samsung, variants, Android-based and IOS behave the same) and only Firefox did mint audio at all times BUT since last update FF doesnt show the mic anymore, so Audio wont record at all. 

Entering a room after a fresh reload OR scrolling up to lood more messages has no impact. 

i tried the following already (after taking a backup) i deleted all medial message of channel -> no diffrence, i deleted alot of messages like 24000 of 26000 total -> no diffrence, i deleted ALL messages of channel -> bingo , clear audio (that might just have to do with what the chat thread does besides recording)

**Comment by zigfighter** [comment]
![Screenshot_20190510-115342](https://user-images.githubusercontent.com/44551315/57524634-abd10880-7328-11e9-834b-7bf128743662.jpg)

**Comment by zigfighter** [comment]
this is from Android, fresh empty phone just FF installed.

**Comment by geekgonecrazy** [comment]
More details...

Can do this on any browser.  Start audio recording and then start doing things that cause a lot of JavaScript to execute.  Expand and collapse panels etc.  then listen to output.  

Current implementation seems to use an interval to record.  My guess is event loop gets some extra tasks in it  and it’s not getting a continuous recording.

**Comment by zigfighter** [comment]
Since last update, this Issue magicly vanished on my installation right now i cant reproduce any of the jittering effecs. i do some more testing now with various browsers on various devices and on wifi/various mobile connects.

**Comment by geekgonecrazy** [comment]
I don’t believe any changes have been made.  Likely just not a lot of load at the time

**Comment by zigfighter** [comment]
Yup, i just reconfirmed it again.. just a fluke,.. its still there :-/
